dissect/target/plugins/os/windows/adpolicy.py

```diff
@@ -78,7 +78,7 @@ def _xmltasks(self, policy_dir):
                         guid=task.attrib.get("uid"),
                         key="XML",
                         value=properties.attrib.get("appName"),
-                        size=len(xml),
+                        size=len(task),
                         data=ElementTree.tostring(task),
```

Comment:
```suggestion
                        task_data = ElementTree.tostring(task)
                        ...
                        size=len(task_data),
                        data=ElementTree.tostring(task_data),
```

Ithink this would be a bit more in line with the .pol file records.